tests/auth_tests/test_tokens.py

```diff
@@ -140,3 +140,39 @@ def test_secret_lazy_validation(self):
         msg = 'The SECRET_KEY setting must not be empty.'
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
             default_token_generator.secret
+
+    def test_verify_with_secret_fallbacks(self):
+        user = User.objects.create_user('tokentestuser', 'test2@example.com', 'testpw')
+        p1 = PasswordResetTokenGenerator()
+        p1.secret = 'secret'
+        tk = p1.make_token(user)
+        p2 = PasswordResetTokenGenerator()
+        p2.secret = 'newsecret'
+        p2.secret_fallbacks = ['secret']
+        self.assertTrue(p1.check_token(user, tk))
+        self.assertTrue(p2.check_token(user, tk))
+
+    @override_settings(
+        SECRET_KEY='secret',
+        SECRET_KEY_FALLBACKS=['old_secret'],
```

Comment:
Let's be consistent, if the previous test we have `newsecret`:
```suggestion
        SECRET_KEY_FALLBACKS=['oldsecret'],
```